src/app/view/components/dot-persona-selector/dot-persona-selector.component.spec.ts

```diff
@@ -18,6 +19,7 @@ describe('DotPersonaSelectorComponent', () => {
     const messageServiceMock = new MockDotMessageService({
         'modes.persona.no.persona': 'Default Persona'
     });
+    const stringPixels = new StringPixels;
```

Comment:
should be a mock

Reply:
this is not needed anymore, so I just removed it